easybuild/framework/easyconfig/default.py

```diff
@@ -165,6 +165,7 @@
 
     # OTHER easyconfig parameters
     'buildstats': [None, "A list of dicts with build statistics", OTHER],
+    'options' : [{},"A list of extra options for specific blocks", OTHER],
```

Comment:
'pass anything to any block' should be avoided. also, custom easyblocks can define their own block-specifc options